### PR TITLE
Feature/id3

### PR DESCRIPTION
**Pull request by dominic-miglar**

### CODE REVIEW

File: bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py [hunk 1]
```diff
@@ -14,7 +13,7 @@
 
 class H264CodecConfiguration(VideoCodecConfiguration, Serializable):
 
-    def __init__(self, name, bitrate, rate, profile, id_=None, description=None, custom_data=None, width=None,
+    def __init__(self, name, bitrate, profile, rate=None, id_=None, description=None, custom_data=None, width=None,
```
Comment (rkersche) [inline bitmovin/resources/models/codecconfigurations/h264_codec_configuration.py]: Do not change this

File: bitmovin/resources/models/codecconfigurations/h265_codec_configuration.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 class H265CodecConfiguration(VideoCodecConfiguration, Serializable):
 
-    def __init__(self, name, bitrate, rate, profile, id_=None, description=None, custom_data=None, width=None,
+    def __init__(self, name, bitrate, profile, rate=None, id_=None, description=None, custom_data=None, width=None,
```
Comment (rkersche) [inline bitmovin/resources/models/codecconfigurations/h265_codec_configuration.py]: Do not change this

File: bitmovin/package_information.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
 NAME = 'bitmovin-python'
-VERSION = '1.6.0'
+VERSION = '1.7.0dev1'
```
Comment (dominic-miglar) [inline bitmovin/package_information.py]: yes, will be one without dev as soon it's released. 
https://www.python.org/dev/peps/pep-0440/#developmental-releases

Comment (moser4035) [inline bitmovin/package_information.py]: Then it's OK

**Pull request merged**